src/services/submit.ts

```diff
@@ -97,10 +99,12 @@ export default async function submit({
         })
       }
 
-      console.log('Offline - saving submission to pending queue..')
+      console.log(
+        'Offline or always submitting via pending queue - saving submission to pending queue..',
+      )
       await addFormSubmissionToPendingQueue(formSubmission)
       return Object.assign({}, formSubmission, {
-        isOffline: true,
+        isOffline: isOffline(),
```

Comment:
should make this a variable when you first call it in case a miracle happens and they're online again when you call it here.

Reply:
I dont think so. If they are online when it gets here we want it to say that in the message. If they are offline, we want it to say that. Ideally the message represents the latest status we know, not the state as it was before.